keyboards/ivy/rev1/rev1.h

```diff
@@ -0,0 +1,22 @@
+#ifndef REV1_H
+#define REV1_H
+
+#include "../ivy.h"
```

Comment:
The `../` isn't necessary.

```suggestion
#include "ivy.h"
```